### PR TITLE
Updating  - with AI formats - AI label and sensitivity label

### PR DESCRIPTION
**Pull request by Jegadeesh-MSFT**

### CODE REVIEW

File: samples/bot-conversation/nodejs/appManifest/manifest.json [hunk 1]
```diff
@@ -58,6 +58,26 @@
...
+              "title": "Feedback buttons",
+              "description": "Feedback buttons that helps to provide feedback for a bot message"
+            },
+            {
+              "title": "citation",
+              "description": "Used to cite the sources of the bot message"
+            },
+            {
+              "title": "SendAITextMessage",
+              "description": "Sends an format AI text message using bot"
```
Comment (premkaretimsft) [inline samples/bot-conversation/nodejs/appManifest/manifest.json:80]: Sends an AI generated message in text format using bot

File: samples/bot-conversation/nodejs/bots/teamsConversationBot.js [hunk 2]
```diff
@@ -111,6 +122,125 @@ class TeamsConversationBot extends TeamsActivityHandler {
...
+                    "@id": "",
+                    citation: [
+                        {
+                            "@type": "Claim",
+                            position: 1, // Required. Should match the [1] in the text above
+                            appearance: {
+                                "@type": "DigitalDocument",
+                                name: "Some secret citation", // Title
+                                url: "https://example.com/claim-1", // Hyperlink on the title
+                                abstract: "Excerpt", // Excerpt (abstract)
```
Comment (premkaretimsft) [inline samples/bot-conversation/nodejs/bots/teamsConversationBot.js:229]: add `encodingFormat` as well

**Review by premkaretimsft — APPROVED** [review]

File: samples/bot-conversation/nodejs/appManifest/manifest.json (hunk 1 above)
Comment (Jegadeesh-MSFT) [reply samples/bot-conversation/nodejs/appManifest/manifest.json:80]: Fixed

File: samples/bot-conversation/nodejs/bots/teamsConversationBot.js (hunk 2 above)
Comment (Jegadeesh-MSFT) [reply samples/bot-conversation/nodejs/bots/teamsConversationBot.js:229]: Fixed

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by Wajeed-msft — APPROVED** [review]

**Comment by Pawank-MSFT** [comment]
Executed build pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/9178251034

**Pull request merged**